checks.py

```diff
@@ -0,0 +1,45 @@
+from models import db, Balls, Bowler, Batsman
```

Comment:
I've checked out this branch
`git checkout add_bowlers_logic` 
And It can't find `Batsman from models`